### DISCUSSION TITLE
不同表上线DDL 走不同的审核节点功能

### DISCUSSION THREAD

**Discussion by turingdream** [hhyo/Archery#1666] [Ideas]
我们这边有一个需求，业务库上线DDL 的审批流程，期望能加入大数据的数仓人员审核，但是需要不同库，不同表，让不同的数仓小团队人员参与审核

**Comment by hhyo** [comment]
审核流程配置到表级别？ 猜测需求的背景应该是数仓有一些规范需要在建表时进行控制，或者是有些优先级高的表他们需要感知变更的影响，这种如果是硬性限制，可能直接从审核规则控制会更合适。如果只是评估影响，数仓可以通过监控DDL变更来感知，数据库侧应该很少会将数据资产归属收集到表维度

**Comment by turingdream** [comment]
审核规则控制，这个我研究下，有资料链接么？
